Dockerfile

```diff
@@ -65,12 +58,41 @@ RUN mkdir /usr/src/couchdb && cd /usr/src/couchdb \
 # compile and install couchdb
 RUN cd /usr/src/couchdb \
   && ./configure -c --disable-docs \
-  && make release \
-  && mv /usr/src/couchdb/rel/couchdb /couchdb
+  && make release
+
+# critical package deps preventing update: openjdk-7-jdk, libnspr4-0d, libicu52
+FROM erlang:18 as ntr-couch-clouseau
+ENV MAVEN_HOME /usr/share/maven
+ENV COUCHDB_PATH /couchdb
```

Comment:
lets use the standard /opt/couchdb and /opt/clouseau path for compatibility with the helm chart, as it that does not allow variable paths i think